lib/phoenix/endpoint.ex

```diff
@@ -449,7 +454,7 @@ defmodule Phoenix.Endpoint do
       end
 
       if var!(config)[:debug_errors] do
-        use Plug.Debugger, otp_app: @otp_app
+        use Plug.Debugger, otp_app: @otp_app, style: @debugger_style
```

Comment:
Can we inline the attributes here in this case? That's because using module attributes first requires the binary to be copied to ETS and then into the compiled module. That step is quite harmless in Plug.Debugger because we compile it only once in the project life time but, because we may define multiple endpoits, we want to avoid this cost.